elements/pfe-tabs/src/pfe-tab-panel.scss

```diff
@@ -70,7 +70,18 @@ $LOCAL-VARIABLES: (focus: pfe-var(link),
 /// HORIZONTAL
 /// ===========================================================================
 :host([variant="earth"]) {
+  background-color: pfe-local(BackgroundColor, $region: panel, $fallback: #{pfe-var(surface--lightest)});
 
+  .container {
+    padding-top: pfe-local(PaddingTop, $region: panel, $fallback: calc(#{pfe-var(container-spacer)} * 3));
```

Comment:
You can remove the fallbacks on the items below since they all match the mapped defaults